### PR TITLE
RT56 Turkey Last Major Update

### PR DESCRIPTION
**Pull request by BiccyThiccy**
- Every political path has been fully reworked from the ground up.
- Localization improvements: the couple of Turkish names have been deleted, I deleted all of the localization collides I could find, and made some improvements to existing stuff.
- Added Thrace as a coreable state (same decision menu as the Azerbaijani stuff).
- Other misc crap in the focus tree, too much to list really.
- RT56 Tree now starts with no cores on Kurdish states, and modifiers akin to what Sudan has.

### For the patch-notes:
**Highlights:**

- The rework to the Road to 56 Turkish Focus Tree has officially been completed, reworking the political paths from the ground up, and breathing life into other areas of the tree.

### CODE REVIEW

**Comment by BiccyThiccy** [comment]
Needs before I mark ready for review:

- Localize the remaining stuff
- Get feedback
- See if there's any last minute additions I can make

as far as I can tell, error log is clean

**Review by marijn211 — COMMENTED** [review]
Dont deviate formatting standards, unless you had a good reason

**Review by marijn211 — DISMISSED** [review]
Rename event options not following a/b/c/d/e naming format, make sure leader events dont create duplicate leader roles like Ismet Inonu now gets

**Review by TheBrit7 — DISMISSED** [review]

**Comment by TheBrit7** [comment]
Seems good, just needs approval from Marijn.

File: common/scripted_triggers/00_scripted_triggers.txt
Comment (ilkober) [inline common/scripted_triggers/00_scripted_triggers.txt:1]: NOT = { AND = { can be replaced

File: events/BFTB_Turkey.txt
Comment (ilkober) [inline events/BFTB_Turkey.txt:1]: No base value in ai_chance meaning that factor will not have an impact there

File: events/Turkey.txt
Comment (ilkober) [inline events/Turkey.txt:1]: Same here

Should be like:
ai_chance = {
       base = X
       modifier = {
             factor = 100
             [YOUR CONDITIONS]
       }
}

File: interface/replace/r56_goals_shine.gfx
Comment (ilkober) [inline interface/replace/r56_goals_shine.gfx:1]: Two icons are defined, while the new shine is defined for only one means that unless it is defined elsewhere, the shine will not appear while cult_of_the_marshal is selected

Comment (BiccyThiccy) [reply interface/replace/r56_goals_shine.gfx:1]: Shine works fine in game as far as I can tell, please check the video I linked in #burkey-turkey

**Review by ilkober — APPROVED** [review]
Merging time
Looks good

**Comment by BiccyThiccy** [comment]
🫡

**Pull request merged**